etc/services/preprints-lissa.json

```diff
@@ -3,7 +3,7 @@
   "id" : 203948234207249,
   "name" : "LISSA",
   "description" : "",
-  "serviceId" : "^https?://(localhost|127\\.0\\.0\\.1):5000/(login|logout)/?\\?next=https?(%3A|:)(%2F|/)(%2F|/)((local\\.lissa\\.(org|com)(%3A|:)4200)|((localhost|127\\.0\\.0\\.1)(%3A|:)5000(%2F|/)preprints(%2F|/)lissa))($|%2F|/).*",
```

Comment:
`lissa.org` should be `lissarchive.org`, OSF/Preprints side update required if this provider wants branded domain.